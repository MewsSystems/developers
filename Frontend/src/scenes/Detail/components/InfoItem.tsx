import React from 'react'
import styled from 'styled-components'

const StyledInfoItem = styled.div`
  font-size: 0.8rem;
`

export interface InfoItemProps {
  label: React.ReactNode | React.ReactNodeArray
  chilren?: React.ReactNode | React.ReactNodeArray
}

export const InfoItem: React.FC<InfoItemProps> = ({ label, children }) => (
  <StyledInfoItem>
    <strong>{label}</strong>
    <div>{children}</div>
  </StyledInfoItem>
)
