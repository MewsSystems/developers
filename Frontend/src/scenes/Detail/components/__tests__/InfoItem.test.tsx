import React from 'react'
import { shallow } from 'enzyme'
import { InfoItem } from '../InfoItem'

describe('Test InfoItem component', () => {
  it('Tests snapshot', () => {
    expect(shallow(<InfoItem label="test" />)).toMatchSnapshot()
  })
})
