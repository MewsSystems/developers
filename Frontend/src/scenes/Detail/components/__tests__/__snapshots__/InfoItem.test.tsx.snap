// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test InfoItem component Tests snapshot 1`] = `
<styled.div>
  <strong>
    test
  </strong>
  <div />
</styled.div>
`;
